CONTRIBUTING.md

```diff
@@ -182,6 +182,13 @@ any of the following conditions:
 [1] - The aim is to reduce the burden of re-ACK'ing trivial changes and also
       alleviate the problem of devs spread around the world in different timezones.
 
+#### Backporting
+
+Backporting patches that have been merged into master for release should be done
```

Comment:
```suggestion
Backporting patches that have been merged into `master` for release should be done
```
In other parts of this document we are using `master` (with backticks).

Reply:
Definitely, thanks man.